x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/new_entity_flyout.cy.ts

```diff
@@ -67,6 +67,14 @@ describe(
         expandRiskInputsFlyoutPanel();
         cy.get(RISK_INPUT_PANEL_HEADER).should('exist');
       });
+
+      it('should show asset criticality in the risk input panel', () => {
+        waitForAlerts();
```

Comment:
NIT: You can move the `waitForAlerts()` of all the tests to the `beforeEach()` hook.